diff --git a/BUILD b/BUILD
index c1e6757ca..023ec97b5 100644
--- a/BUILD
+++ b/BUILD
@@ -69,10 +69,11 @@ alias(
 )
 
 # The directory structure for distribution
+exports_files(["typedb_server_bin.exe"])
 pkg_files(
     name = "package-layout-server",
-    srcs = ["//:typedb_server_bin", "//binary:typedb"],
-    renames = {"//:typedb_server_bin" : "server/typedb_server_bin"},
+    srcs = ["//:typedb_server_bin.exe", "//binary:typedb.bat"],
+    renames = {"//:typedb_server_bin.exe" : "server/typedb_server_bin.exe"},
     attributes = binary_permissions,
 )
 
