diff --git a/BUILD b/BUILD
index 0cbdcc5da..c363ea01c 100644
--- a/BUILD
+++ b/BUILD
@@ -24,23 +24,10 @@ exports_files(
     ["VERSION", "deployment.bzl", "LICENSE", "README.md"],
 )
 
-rust_binary(
+exports_files(["typedb_server_bin.exe"])
+alias(
     name = "typedb_server_bin",
-    srcs = [
-        "main.rs",
-    ],
-    deps = [
-        "//common/logger",
-        "//database",
-        "//resource",
-        "//server",
-
-        "@crates//:clap",
-        "@crates//:tokio",
-    ],
-    compile_data = [
-        ":VERSION",
-    ],
+    actual = ":typedb_server_bin.exe",
 )
 
 # Assembly
