diff --git a/BUILD b/BUILD
index 8a126f224..46feaff4f 100644
--- a/BUILD
+++ b/BUILD
@@ -71,9 +71,9 @@ alias(
 # The directory structure for distribution
 pkg_files(
     name = "package-layout-server",
-    srcs = ["//:typedb_server_bin", "//binary:typedb", "//server:config.yml"],
+    srcs = ["//:typedb_server_bin.exe", "//binary:typedb.bat", "//server:config.yml"],
     renames = {
-        "//:typedb_server_bin" : "server/typedb_server_bin",
+        "//:typedb_server_bin.exe" : "server/typedb_server_bin.exe",
         "//server:config.yml" : "server/config.yml",
     },
     attributes = binary_permissions,
