diff --git a/BUILD b/BUILD
index a0be2b37e7..1aa5de414a 100644
--- a/BUILD
+++ b/BUILD
@@ -75,13 +75,23 @@ alias(
     })
 )
 
-# The directory structure for distribution (Unix)
+filegroup(
+    name = "typedb_server_bin_release",
+    srcs = ["typedb_server_bin.exe"],
+)
+
+filegroup(
+    name = "typedb_admin_bin_release",
+    srcs = ["typedb_admin_bin.exe"],
+)
+
+# The directory structure for distribution (Windows)
 pkg_files(
     name = "package-layout-server",
-    srcs = ["//:typedb_server_bin", "//admin:typedb_admin_bin", "//binary:typedb", "//server:config.yml", "//:LICENSE"],
+    srcs = ["//:typedb_server_bin_release", "//:typedb_admin_bin_release", "//binary:typedb.bat", "//server:config.yml", "//:LICENSE"],
     renames = {
-        "//:typedb_server_bin" : "server/typedb_server_bin",
-        "//admin:typedb_admin_bin" : "admin/typedb_admin_bin",
+        "//:typedb_server_bin_release" : "server/typedb_server_bin.exe",
+        "//:typedb_admin_bin_release" : "admin/typedb_admin_bin.exe",
         "//server:config.yml" : "server/config.yml",
     },
     attributes = binary_permissions,
@@ -145,7 +155,7 @@ select_file(
     name = "console-binary-only",
     srcs = ":typedb_console_artifact_extracted",
 #     strip_prefix = strip_prefix.from_pkg("typedb-console-mac"),
-    subpath = "console/typedb_console_bin",
+    subpath = "console/typedb_console_bin.exe",
 )
 
 pkg_files(
