diff --git a/BUILD b/BUILD
index dd59a2ef3..68d21af7d 100644
--- a/BUILD
+++ b/BUILD
@@ -80,10 +80,10 @@ alias(
 # The directory structure for distribution
 pkg_files(
     name = "package-layout-server",
-    srcs = ["//:typedb_server_bin", "//admin:typedb_admin_bin", "//binary:typedb", "//server:config.yml"],
+    srcs = ["//:typedb_server_bin.exe", "//admin:typedb_admin_bin.exe", "//binary:typedb.bat", "//server:config.yml"],
     renames = {
-        "//:typedb_server_bin" : "server/typedb_server_bin",
-        "//admin:typedb_admin_bin" : "admin/typedb_admin_bin",
+        "//:typedb_server_bin.exe" : "server/typedb_server_bin.exe",
+        "//admin:typedb_admin_bin.exe" : "admin/typedb_admin_bin.exe",
         "//server:config.yml" : "server/config.yml",
     },
     attributes = binary_permissions,
